ckan/tests/controllers/test_package.py

```diff
@@ -2051,6 +2052,77 @@ def test_create_dataset(self, app):
         assert dataset["id"] in href.select_one("a")["href"].split("/", 2)[-1]
         assert dataset["title"] in href.text.strip()
 
+    @pytest.mark.ckan_config("ckan.activity_list_limit", "3")
```

Comment:
These are great but can you add a couple more tests to ensure that:
* Navigating to the activity page using `before` and `after` actually works
* Passing a non-timestamp value returns an error (or ignores the params)
* Previous button is not shown in the first page
* Next button is not shown in the last page